src/components/templates/Playa/Router.tsx

```diff
@@ -1,16 +1,19 @@
 import React from "react";
 import { useRouteMatch, Switch, Route } from "react-router-dom";
-import Playa from ".";
-import PlayaAdmin from "./PlayaAdmin";
+
 import useConnectCurrentVenue from "hooks/useConnectCurrentVenue";
 
+import { BannerAdmin } from "components/organisms/BannerAdmin";
+
+import Playa from ".";
```

Comment:
I thought Playa is completely removed from our codebase?

Reply:
See https://github.com/sparkletown/sparkle/pull/1080#discussion_r564114898